src/utils/env.ts

```diff
@@ -109,7 +109,7 @@ export function readEnvironmentConfig(overwriteExisting = false): void {
   let newEnv = {} as any;
 
   // Read from home dir
-  const homedirEnvFile = join(homedir(), ENV_FILE_NAME);
```

Comment:
These are all automated linting changes, right? Would've been better to do a separate PR for these so that the material changes are obvious.